packages/contracts/scripts/engine/V3/generic-engine-shared-suite-deployer.ts

```diff
@@ -482,8 +481,15 @@ async function main() {
     }
 
     // create image bucket
-    const { bucketName } = await createEngineBucket(tokenName, networkName);
-    console.log(`[INFO] Created image bucket ${bucketName}`);
+    let imageBucketCreated = false;
+    let bucketName = "TBD";
```

Comment:
Was this left in unintentionally?

Reply:
ah - yes, but good callout for a dev comment. added clarifying dev comments in [5da9a82](https://github.com/ArtBlocks/artblocks-contracts/pull/964/commits/5da9a8234f6cfff5b7973e16a23dbe2184568514)